content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,595 @@
+{{< /hint >}}
+
+The `crossplane beta render` command connects to the locally running Docker 
+Engine to pull and run composition functions. 
+
+{{<hint "important">}} 
+Running `crossplane beta render` requires [Docker](https://www.docker.com/).
+{{< /hint >}}
+
+Provide a composite resource, composition and composition function YAML 
+definition with the command to render the output locally. 
+
+For example, 
+`crossplane beta render xr.yaml composition.yaml function.yaml`
+
+The output includes the original composite resource followed by the generated 
+managed resources. 
+
+{{<expand "An example render output" >}}
+```yaml
+---
+apiVersion: nopexample.org/v1
+kind: XBucket
+metadata:
+  name: test-xrender
+status:
+  bucketRegion: us-east-2
+---
+apiVersion: s3.aws.upbound.io/v1beta1
+kind: Bucket
+metadata:
+  annotations:
+    crossplane.io/composition-resource-name: my-bucket
+  generateName: test-xrender-
+  labels:
+    crossplane.io/composite: test-xrender
+  ownerReferences:
+  - apiVersion: nopexample.org/v1
+    blockOwnerDeletion: true
+    controller: true
+    kind: XBucket
+    name: test-xrender
+    uid: ""
+spec:
+  forProvider:
+    region: us-east-2
+```
+{{< /expand >}}
+
+#### Flags
+
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                             | Description                                           |
+| ------------ | -------------                         | ------------------------------                        |
+|              | `--context-files=<key>=<file>,<key>=<file>`    | A comma separated list of files to load for function "contexts." |
+|              | `--context-values=<key>=<value>,<key>=<value>` | A comma separated list of key-value pairs to load for function "contexts."                                                    |
+| `-r`         | `--include-function-results`          | Include the "results" or events from the function.   |
+| `-o`         | `--observed-resources=<directory or file>`               | Provide artificial managed resource data to the function.                                                    |
+|              | `--timeout=`                          | Amount of time to wait for a function to finish.                    |
+{{< /table >}}
+
+The `crossplane beta render` command relies on standard 
+[Docker environmental variables](https://docs.docker.com/engine/reference/commandline/cli/#environment-variables) 
+to connect to the local Docker engine and run composition functions. 
+
+
+#### Provide function context
+
+The `--context-files` and `--context-values` flags can provide data 
+to a function's `context`.  
+The context is JSON formatted data.
+
+#### Include function results
+
+If a function produces Kubernetes events with statuses use the 
+`--include-function-results` to print them along with the managed resource 
+outputs. 
+
+#### Mock existing resources
+
+Provide mocked, or artificial data representing a managed resource with 
```

Comment:
would it help to include the words "existing" and/or "observed" in this sentence to further describe what these mocked resources represent?